src/main/java/com/rtbhouse/kafka/workers/api/WorkersConfig.java

```diff
@@ -187,6 +194,11 @@
                         QUEUE_TOTAL_MAX_SIZE_BYTES_DEFAULT,
                         Importance.MEDIUM,
                         QUEUE_TOTAL_MAX_SIZE_BYTES_DOC)
+                .define(QUEUE_RESUME_RATIO,
```

Comment:
Please add some validation, what if desired ratio exceeds 1.0 or when it is negative?